src/main/java/edu/kis/powp/jobs2d/drivers/ImprovedLoggerDriver.java

```diff
@@ -0,0 +1,35 @@
+package edu.kis.powp.jobs2d.drivers;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+
+import java.util.logging.Logger;
+
+public class ImprovedLoggerDriver implements Job2dDriver {
+    private final Logger logger = Logger.getLogger("global");
+
+    private static long setPositionCount = 0;
+    private static long operateToCount = 0;
```

Comment:
It would be good to have getters for those fields. One could request (on demand) to log those values.